.github/workflows/ci.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           cache: true
       - uses: goreleaser/goreleaser-action@v6
         with:
-          version: latest
+          version: "~> v2"
```

Comment:
I was updated to specify the version of GoReleaser in `.goreleaser.yml` at [this time](https://github.com/aereal/jsondiff/pull/40). So, v2 should guarantee operation.
https://github.com/aereal/jsondiff/blob/0df54bc9043c0e932f10c29289d76ce94a53738c/.goreleaser.yml#L2